deployments/kubernetes/chart/reloader/values.yaml

```diff
@@ -67,6 +67,8 @@ reloader:
     #     cpu: "10m"
     #     memory: "128Mi"
     resources: {}
+    pod:
```

Comment:
@rjshrjndrn kindly add these value in this file too: `Reloader/deployments/kubernetes/templates/chart/values.yaml.tmpl`.